src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css'
+import App from './App';
```

Comment:
Organise imoorts

import React from 'react';
import ReactDOM from 'react-dom/client';

// Styles
import './index.css'

// Components
import App from './App';

We follow a convention of organising imports for readability of our codebase